enos/modules/alb_boundary/main.tf

```diff
@@ -0,0 +1,69 @@
+resource "aws_security_group" "boundary_ingress_sg" {
+  name   = "${var.lb_name_suffix}-server-lb"
+  vpc_id = var.vpc_id
+
+  # Boundary
+  ingress {
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = var.cidr_blocks
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+resource "aws_lb" "boundary_clients_ingress" {
+  name               = "boundary-ingress-alb"
```

Comment:
does this name need to be unique. Like what if there is more than one deployed boundary cluster?

Reply:
The ALB should have an ID that makes it distinguishable. Do you think we should also add a random resource to additionally name this in a unique way?

Reply:
We decided to tag all resources that could be, and this has been done in the new commit.